tests/integration-tests/tests/networking/test_security_groups/test_overwrite_sg/pcluster.config.update_sg.yaml

```diff
@@ -0,0 +1,49 @@
+Image:
+  Os: {{ os }}
+HeadNode:
+  InstanceType: {{ instance }}
+  Networking:
+    SubnetId: {{ public_subnet_ids[0] }}
+    {% if additional_security_groups == False %}
+    SecurityGroups:
+      - {{ vpc_security_group_id[1] }}
+    {% else %}
```

Comment:
`SecurityGroups` and `AdditionalSecurityGroups` can be used together. 

So  if I'm not wrong we can have:
```
pcluster.config.yaml

  Networking:
        SecurityGroups:
          - {{ vpc_security_group_id[0] }}
```


```
pcluster.config.update.yaml


  Networking:
        SecurityGroups:
          - {{ vpc_security_group_id[1] }}
        AdditionalSecurityGroups:
          - {{ vpc_security_group_id[0] }}
```

In this way you can test at the same time what happens updating Security or AdditionalSecurityGroups.
feel free to find better alternatives anyway the goal is to avoid to execute 2 times the update command.

Reply:
I tried that initially. But apparently they both should not [co-exist](https://github.com/aws/aws-parallelcluster/blob/0b79cf32a1bc01d19a4295e8592a8b9cd4ae9dc7/cli/src/pcluster/schemas/cluster_schema.py#L653-L654)
https://github.com/aws/aws-parallelcluster/blob/0b79cf32a1bc01d19a4295e8592a8b9cd4ae9dc7/cli/src/pcluster/schemas/cluster_schema.py#L653-L654